src/arutils.py

```diff
@@ -438,7 +524,42 @@ def polyval2d(x, y, m):
     return z
 
 
-def gauss_lsqfit(x, y, pcen):
+def moffat(x,p0,p1,p2):
+    """  Moffat profile
+    This 3 parameter formulation assumes the trace is known
+    Parameters
+    ----------
+    x
+    p0 : float
+      Amplitude
+    p1 : float
+      Width scaling
+    p2 : float
+
+    Returns
+    -------
+    Evaluated Moffat
+    """
+    return p0 / (1+(x/p1)**2)**p2
```

Comment:
Yes, I think you are right.  
Better to make actual changes to the trace if we want (possible).

Modifying..